Source/ui/UI.js

```diff
@@ -2,9 +2,9 @@
 
 import { isNativeObject } from '../dom/utils'
 
-function getPluginIcon() {
+function getPluginAlertIcon() {
   if (__command.pluginBundle() && __command.pluginBundle().icon()) {
```

Comment:
should check for `alertIcon` method instead?

Reply:
Excellent catch, thanks.